### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by xphysnerd** [jasp-stats/jasp-issues#3009]
### JASP Version

0.19.1

### JASP Module

ANOVA

### What analysis are you seeing the problem on?

Repeated Measures ANOVA

### What OS are you seeing the problem on?

Windows 10

### Bug Description

Results from a previous JASP version differ when looking at Post-Hoc results with a Bonferroni correction. I have run the same data in R and the results match the previous JASP version. Therefore, I am fairly confident that the new JASP version is giving incorrect p-value and CI results.

### Expected Behaviour

Results from a previous JASP version differ when looking at Post-Hoc results with a Bonferroni correction. I have run the same data in R and the results match the previous JASP version. Therefore, I am fairly confident that the new JASP version is giving incorrect p-value and CI results.

### Steps to Reproduce

1.Run a repeated measures ANOVA
2.Perform post-hoc tests with bonferroni correction
3.Run the same analysis on a previous JASP version or in R 
...

### Final Checklist

- [ ] I have included a screenshot showcasing the issue, if possible.
- [ ] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by JohnnyDoorn** [comment]
Hi @xphysnerd ,

Could you please include a data example with the results? We have some safeguards in place that should prevent this kind of stuff, and I am not aware of any changes to the posthoc tests, so any demonstration will greatly help.

Kind regards,
Johnny

**Comment by xphysnerd** [comment]
The current version incorrectly gives the following table:
Post Hoc Tests
Post Hoc Comparisons - Time
95% CI for Mean Difference
Mean Difference Lower Upper SE t pbonf
1 2 1.493 -0.152 3.138 0.672 2.224 0.088
  3 2.987 0.454 5.519 1.034 2.889 0.015
2 3 1.493 -0.228 3.215 0.703 2.125 0.111
*Note.*  P-value and confidence intervals adjusted for comparing a family
of 3 estimates (confidence intervals corrected using the bonferroni method).

The correct tale should be:
[image: image.png]

**Comment by tomtomme** [comment]
Thx for the Feedback. Can you share the data that belong to this result and also the correct R results? Without this I think we have no chance of understanding whats going on. Thx

**Comment by xphysnerd** [comment]
I attached the CSV to the previous message.

**Comment by tomtomme** [comment]
We cannot see Mail Attachments. Pls drag and drop the files Here directly

**Comment by xphysnerd** [comment]
[LDL Data V2.csv](https://github.com/user-attachments/files/18338376/LDL.Data.V2.csv)

**Comment by tomtomme** [comment]
forget what I wrote before. I can now replicate your result:

![grafik](https://github.com/user-attachments/assets/913c28b2-7a99-416d-b03c-4b62dd3fa425)

So what result would you expect? Maybe this one?:
![grafik](https://github.com/user-attachments/assets/88b1c9b0-547b-4fed-9d25-b80ce57ef394)

**Comment by JohnnyDoorn** [comment]
Yes that option could be causing a difference in results, but having the unpooled term is generally recommended (and JASP's default behavior now) since it handles possible sphericity violations better in RM ANOVA. The RM ANOVA results have been verified using Andy Field's statistics books (JASP version out soon!).

**Comment by xphysnerd** [comment]
@tomtomme , yes that yields the typical results that match what most other statistical packages yield for Bonferroni post-hocs.

**Comment by xphysnerd** [comment]
Although non-pooled error may be more flexible, and probably better in most cases, it may be problematic as a default setting for software that is largely used by people who don’t have a deep understanding of stats. People will run their data using the default setting and most users will not describe their model as using non-pooled error in their statistical analysis manuscript sections. I don’t use them but are SAS or SPSS using the in-pooled default? I know that emmeans in R does not default to use non-pooled errors for Bonferroni post-hocs.
